README.md

```diff
@@ -9,14 +9,28 @@
  
 </div>
 
-_For the installation instructions, [click here](#installation)._
```

Comment:
IMO, the first thing that the user looks for when opening a repo is how to install it. This line serves that very purpose.

Reply:
I know, but now they can do it quickly from the Contents. Do you insist to have a dedicated line for it in the first place?

Reply:
The link in the contents is visually buried between the other links and does not immediately stand out, unlike the dedicated line.

Reply:
Ok, moved it back.